README.md

```diff
@@ -191,6 +191,56 @@ boolean, source: HTMLElement}` syntax, both `showPopover()` and
 - The polyfill will not work in browsers with partial popover support enabled,
   and will also not attempt to make experimental support match the final spec.
 
+- Native `popover` elements contain a `::backdrop` pseudo-element when open.
+  This behavior has not been added to the polyfill. To work around this, one option
+  is to add a class to the `body` element when a `popover` is open:
+
+  ```js
+  const popovers = document.querySelectorAll('[popover]');
+  const body = document.getElementById('body');
+
+  if (!isSupported) {
```

Comment:
```suggestion
  if (isPolyfilled()) {
```

Reply:
Yeah, I think it does. There's a potential race condition if this runs before the polyfill runs, but I think it makes sense to check if it's actually applied here.